client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -90,173 +104,182 @@ export const StockLineForm: FC<StockLineFormProps> = ({
+                  >
+                    <Box>
+                      <IconButton
+                        disabled={isScanning || !isConnected}
+                        onClick={scanBarcode}
+                        icon={
+                          isScanning ? (
+                            <CircularProgress size={20} color="secondary" />
+                          ) : (
+                            <ScanIcon />
+                          )
+                        }
+                        label={t('button.scan')}
+                      />
+                    </Box>
+                  </Tooltip>
+                )}
+              </Box>
             }
-            text={String(isPackVariantsEnabled ? packUnit : draft.packSize)}
+          />
+          <TextWithLabelRow
+            label={t('label.supplier')}
+            text={String(supplierName)}
             textProps={{ textAlign: 'end' }}
           />
-        )}
-        <StyledInputRow
-          label={t('label.on-hold')}
-          Input={
-            <Checkbox
-              checked={draft.onHold}
-              onChange={(_, onHold) => onUpdate({ onHold })}
-            />
-          }
-        />
-        <StyledInputRow
-          label={t('label.location')}
-          Input={
-            <LocationSearchInput
-              autoFocus={false}
-              disabled={false}
-              selectedLocation={location}
-              width={160}
-              onChange={location => {
-                onUpdate({ location, locationId: location?.id });
-              }}
-            />
-          }
-        />
-        <StyledInputRow
-          label={t('label.barcode')}
-          Input={
-            <Box display="flex" style={{ width: 162 }}>
-              <BufferedTextInput
-                value={draft.barcode ?? ''}
-                onChange={e => onUpdate({ barcode: e.target.value })}
-              />
-              {isEnabled && (
-                <Tooltip
-                  title={isConnected ? '' : t('error.scanner-not-connected')}
-                >
-                  <Box>
-                    <IconButton
-                      disabled={isScanning || !isConnected}
-                      onClick={scanBarcode}
-                      icon={
-                        isScanning ? (
-                          <CircularProgress size={20} color="secondary" />
-                        ) : (
-                          <ScanIcon />
-                        )
-                      }
-                      label={t('button.scan')}
-                    />
-                  </Box>
-                </Tooltip>
-              )}
-            </Box>
-          }
-        />
-        <TextWithLabelRow
-          label={t('label.supplier')}
-          text={String(supplierName)}
-          textProps={{ textAlign: 'end' }}
-        />
+        </Grid>
       </Grid>
-    </Grid>
+      {footerProps && <Footer {...footerProps} />}
```

Comment:
Normally the Footer is rendered in with the DetailView component, but here we only want the Footer buttons on the main Details tab, so rendering it here instead. It's conditional, as we don't display it when this form appears in Modal.

Reply:
I think it would be better to leave the footer in the DetailView component, and just keep the buttons disabled on the other tabs?

I don't think hiding it from view on the other tabs adds much value, and conditionally rendering a footer from a reusable form component definitely seems wrong 😅

Reply:
Okay, have moved back, but hidden on other tabs -- seems okay to me.